### ISSUE TITLE
Keyboard shortcut "go to Subgroup" or show subgroup path as breadcrumbs to find adjacent entries

### ISSUE THREAD

**Issue by Redsandro** [keepassxreboot/keepassxc#10130]
## Summary

Sometimes you cannot find the entry you are looking for, but you find entries you know are from the same group. So you must manually browse that containing group. Currently, KeepassXC shows the entry subgroup path, but to see the entries in that group, you have to either:

1. (keyboard) Ctrl+F `g:NameOfSubGroup`; or
2. (mouse) manually click and expand every group in the tree. 

It would be a welcome shortcut to be able to browse that group immediately. For example by introducing:

1. a simple keyboard shortcut _"go to group"_; or perhaps more intuitively
2. make the current subgroup path into breadcrumbs i.e. clickable buttons or links to open/browse the subgroup

## Examples

Currently: 
![image](https://github.com/keepassxreboot/keepassxc/assets/1702193/8b54ae94-d0e1-4a10-b33f-610028989f10)

Buttons:
![image](https://github.com/keepassxreboot/keepassxc/assets/1702193/e245d971-2a44-4a0a-8cf5-437821440c73)

Links:
![image](https://github.com/keepassxreboot/keepassxc/assets/1702193/ce3a30cb-07a8-4fe2-a870-dec1f256486f)

Rough mockup but you get the idea.

## Context

This is my biggest recurring workflow bottleneck in KeePass. I suggested a similar but more involved solution three years ago: https://github.com/keepassxreboot/keepassxc/issues/5423

**Comment by droidmonkey** [comment]
When searching those "related" entries, just double click the group name in the search results to open that group. 

I do like the idea of making the breadcrumbs traversable, but that would be a major UI change. As of right now it is a simple text label.

**Comment by Redsandro** [comment]
> double click the group name in the search results to open that group.

I did not know this. I 100% thought double clicking on the row would open the entry dialog. I guess I always clicked on the title in the row.

This secret knowledge is very near to the desired result, and quite helpful. I probably wouldn't have opened this feature request, had I discovered it sooner through more adventurous and rigorous experimentation or reading of the docs.

While breadcrumbs or in-label links would visually communicate its function thus making it more intuitive for others, selfishly I am satisfied and I will close this issue because I didn't mean to propose a major change.
